src/program.js

```diff
@@ -210,6 +210,13 @@ Example: $0 --help run.
           demand: true,
           type: 'string',
         },
+        'api-proxy': {
+          describe: 
```

Comment:
Ah, I turned off Eslint earlier today then forgot to turn it on, so didn't even notice the white space. All fixed now, thanks for your patience!